### PR TITLE
Specifying mailer servers when possible and tweaking dynamic parts

### PR DESCRIPTION
**Pull request by weaverryan**
| Q             | A
| ------------- | ---
| License       | MIT

1) Instead of just `@ses` I've tried to add the real host. I didn't know all of them, so I left those alone for now. I'd love for someone to test the missing `http` ones and update them accordingly.

2) I changed the syntax for the parts that the user is supposed to update - I thought using the `$` might make a developer think this was some sort of variable. I also added `YOUR-` at the beginning as (hopefully) a queue that they should be *their* information there.

Cheers!

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
$VAR are real references, not just for-the-doc placeholders. Not sure this is better.

**Comment by weaverryan** [comment]
> $VAR are real references,

You mean they are actually variables of some sort? Or am I misunderstanding?

File: symfony/sendgrid-mailer/4.3/manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "env": {
-        "#1": "MAILER_DSN=smtp://$SENDGRID_KEY@sendgrid"
+        "#1": "MAILER_DSN=smtp://apikey:{YOUR-SENDGRID-API-KEY}@smtp.sendgrid.net"
```
Comment (weaverryan) [inline symfony/sendgrid-mailer/4.3/manifest.json]: Actually, I think I'm misunderstanding this too - the `apikey` username looks like it should be provided by the transport class: https://github.com/symfony/sendgrid-mailer/blob/master/Smtp/SendgridTransport.php#L29

**Comment by weaverryan** [comment]
Closing - I may propose some tweaks, but a lot of this was me misunderstanding things... as usual :p

**Pull request closed without merging**